lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -77,25 +78,41 @@ $description = Get-AnsibleParam -obj $params -name "description" -type "str" -de
 $enabled = Get-AnsibleParam -obj $params -name "enabled" -type "bool" -default $true
 $path = Get-AnsibleParam -obj $params -name "path" -type "str" -default '\'
 
+# Ensure the TaskPath starts with and ends with slashes as this causes issues with some
+# versions of Windows
+if ($path[0] -ne '\') {
```

Comment:
@DepthDeluxe Not sure what you are getting at. What I am saying is that I want to force the user to provide an absolute path only. If it does not start with a slash, it should fail with an error. Not pre-pend a slash ourselves.